lib/bloc/tasks_cubit/tasks_cubit.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:bloc/bloc.dart';
+import 'package:meta/meta.dart';
+
+part 'tasks_state.dart';
+
+class TasksCubit extends Cubit<TasksState> {
+  TasksCubit() : super(TasksInitial());
+
+  List<String> tasksList = [];
```

Comment:
This is a minor thing but anyway:
1. Either call this variable tasks (without List in the end)
2. Or call it taskList
Postfix List already mean that there are multiple tasks in it, no need to write taskS.